src/pages/guides/eqp/v1/sandbox.md

```diff
@@ -36,5 +36,5 @@ The following list highlights some of the key differences between the sandbox an
 -  [Session tokens](auth.md#how-to-use-a-session-token) that are generated in the **sandbox** are separate and distinct from **production**. You cannot use session tokens from one environment in the other environment.
```

Comment:
Update image on this page with latest UI. We can use the following: 

![image](https://github.com/AdobeDocs/commerce-marketplace/assets/44397551/deb03271-b035-4e72-8ddd-7fd5b6ec4184)